FirebaseFunctions/Sources/Internal/FunctionsSerializer.swift

```diff
@@ -86,7 +86,7 @@ class FUNSerializer: NSObject {
       }
 
       let decoded = NSMutableDictionary()
-      var decodeError: Error?
+      var decodeError: (any Swift.Error)?
```

Comment:
It could, but we would then need to downcast the error in the `catch` block below, so we won’t gain much in clarity.

What we can do instead is get rid of this Objective-C-inspired construction altogether and rethrow the error from the closure without touching it.